watson/cli.py

```diff
@@ -971,3 +971,40 @@ def merge(watson, frames_with_conflict, force):
     watson.frames = original_frames
     watson.frames.changed = True
     watson.save()
+
+
+@cli.command()
+@click.option('--project', default=None, nargs=2)
+@click.option('--tag', default=None, nargs=2)
+@click.pass_obj
+def rename(watson, project, tag):
+    # input validation
+    if not tag and not project:
+        raise click.ClickException(style('error', 'You have to call rename with --project or --tag'))
+
+    if tag:
+        old_tag, new_tag = tag
+        if old_tag not in watson.tags:
+            raise click.ClickException(style('error', 'Tag "%s" does not exist' % old_tag))
+
+    if project:
+        old_project, new_project = project
+
+        if old_project not in watson.projects:
+            raise click.ClickException(style('error', 'Project "%s" does not exist' % old_project))
+
+    # rename project/tag
+    for frame in watson.frames:
+        watson.frames[frame.id] = (
```

Comment:
Could you use something like:
```python
if frame.project == old_project:
    frame._replace(frame=new_project)
```

That way if we change the format of the frame we will not have to edit this function.